.changeset/cyan-papayas-count.md

```diff
@@ -0,0 +1,7 @@
+---
+"@logto/console": patch
+"@logto/schemas": patch
+"@logto/core": patch
+---
+
+unify image upload file size limit to 500KB on both frontend and backend
```

Comment:
The changeset does not reflect the current change 100% since we also update the max upload file size to be 10MB?